lib/wrap-queries.js

```diff
@@ -19,8 +29,10 @@ function wrap() {
 
       // e.g. print.css
       if (name === 'print') {
+        hasAtRule(src);
         contents = new Buffer('\n@media print {\n' + src + '}');
       } else if (!isNaN(startingNumber)) {
+        hasAtRule(src);
```

Comment:
Warning, and then passing through the file, would certainly be easier. I'm having a hard time finding a clear example of how I toss and error here in a way that's standalone, but can still be caught in the test.